### ISSUE TITLE
Investigating Why Contrastive Learning Benefits Robustness against Label Noise

### ISSUE THREAD

**Issue by HYUNMIN-HWANG** [One-Day-One-Paper/One-Day-One-Paper#73]
# Investigating Why Contrastive Learning Benefits Robustness against Label Noise
self-supervised learning 방법 중 Contrastive learning 의 성능이 좋다는 것이 알려지고 있지만, 어떤 원리에 의해서 작동하는지에 대해서는 많이 밝혀진 바 없다. 해당 논문에서 저자는 
1) one prominent singular value corresponding to every sub-class in the data, and remaining significantly smaller singular values;
2) a large alignment between the prominent singular vector and the clean labels of each subclass.             

위 두 가지 이유로 Contrastive leaning의 성능이 좋다고 가정함

결과적으로 서로 다른 class 들에 대한 에러를 줄이고, noisy label 들을 잘 구분할 수 있음, 그리고 overfitting을 속도를 감소, robustness를 향상시킴

---
* **self supervised contrastive learning** : learns representations by maximizing agreement between differently augmented views of the same data point via a contrastive loss in the latent space
* 결과 비교
![image](https://user-images.githubusercontent.com/70581043/152681584-35f8ff29-6084-44ce-8141-446b6e2b138d.png)       

 (a) distribution of singular values of the Jacobian at initialization
   - pre-trained의 singular values가 급격히 줄어듬      

 (b) alignment of the clean label vector with the Jacobian
   - pre-training이 그다지 성능이 좋아지지는 않았다.       

 (c) test accuracy of fine-tuning a pre-trained network vs. that of a randomly initialized and a linear model trained on representations learned by contrastive learning
   - pre-trainined가 다른 것들보다 성능이 더 좋다      

 (d) loss on noisy labeled data points.
   - pre-trainined가 overfit하기 까지 시간이 더 걸린다.
